src/jit/optimizer.cpp

```diff
@@ -7038,15 +7038,6 @@ void Compiler::optHoistLoopBlocks(unsigned loopNum, ArrayStack<BasicBlock*>* blo
                         m_beforeSideEffect = false;
                     }
                 }
-                else if (tree->OperIsCopyBlkOp())
-                {
```

Comment:
I think It would be better if we call a helper that handles the general case, but I did not find any other places where we check this condition, so I am OK with that change.